src/NativeScript/inspector/InspectorPageAgent.mm

```diff
@@ -38,7 +40,18 @@
 }
 
 void InspectorPageAgent::reload(ErrorString&, const bool* in_ignoreCache, const String* in_scriptToEvaluateOnLoad) {
-    ASSERT_NOT_REACHED();
+    JSValue liveSyncCallback = m_globalObject->get(m_globalObject.globalExec(), Identifier::fromString(&m_globalObject.vm(), "__onLiveSync"));
+    if (!liveSyncCallback.isUndefined()) {
```

Comment:
You don't need this check - `getCallData` already checks that the value is a Cell.